web/themes/custom/theme_server/theme_server.libraries.yml

```
@@ -0,0 +1,31 @@
+global-styling:
+  version: 1.0
+  css:
+    theme:
+      dist/css/style.css: {}
+
+global-scripts:
+  js:
+    js/scripts.js: {}
```

```suggestion
    src/js/scripts.js: {}
```

But:

1. Lets comment out this section, so it doesn't load that JS file.

2. Lets add to scripts.js

```
/**
 * @file
 */

(function ($, Drupal) {

  /**
   * Your script goes here.
   */
  Drupal.behaviors.themeServerGeneral = {
    attach: function (context, settings) {
    }
  };
})(jQuery, Drupal);

```